hacker/views.py

```diff
@@ -1,3 +1,61 @@
-from django.shortcuts import render
+from django.conf import settings
+from django.contrib.auth import authenticate, get_user_model, login
+from django.contrib.auth.decorators import login_required, user_passes_test
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.shortcuts import redirect, render, render_to_response
+from django.urls import reverse_lazy
+from django.utils.decorators import method_decorator
+from django.views import generic as generic_views
+from django.views.generic import base as base_views
 
-# Create your views here.
+from hacker import forms as hacker_forms
+from hacker import models as hacker_models
+
+
+def email_confirmation_check(user):
+    return user.has_confirmed_email() 
+
+def has_application_check(user):
+    return user.has_related_application()
+
+def has_confirmation_check(user):
+    return user.has_related_confirmation()
+
+def has_team_check(user):
+    return user.has_related_team()
+
+
+class dashboard(LoginRequiredMixin):
```

Comment:
classes always start with a capital letter.
```suggestion
class Dashboard(LoginRequiredMixin):
```